tst/RtcpFunctionalityTest.cpp

```diff
@@ -304,24 +304,24 @@ static void parseTwcc(const std::string& hex, const uint32_t expectedReceived, c
     rtcpPacket.header.packetLength = payloadLen / 4;
     rtcpPacket.payload = payload;
     rtcpPacket.payloadLength = payloadLen;
-    KvsPeerConnection connection{};
-    std::pair<UINT32, UINT32> pair{};
-    connection.onPacketNotReceivedCustomData = reinterpret_cast<UINT64>(&pair);
-    connection.onPacketReceivedCustomData = reinterpret_cast<UINT64>(&pair);
-    connection.onPacketReceived = [](UINT64 pair64, UINT16 /*unused*/, INT32 /*unused*/) {
-        auto* pair = reinterpret_cast<std::pair<UINT32, UINT32>*>(pair64);
-        pair->first++;
-    };
+    TwccManager twcc{};
```

Comment:
Not sure what is causing the failure of the tests on Windows. Since it is failing only in WIndows and it could an instability with the platform TravisCI offer, I would suggest you to update the `.travis.yml` to exclude running these tests on Windows for the time being.

Reply:
windows build fails on tests i havent touched, which is double weird